ledger/sandbox/src/main/scala/com/digitalasset/platform/index/JdbcIndex.scala

```diff
@@ -26,11 +26,8 @@ object JdbcIndex {
     ReadOnlySqlLedger(jdbcUrl, Some(ledgerId), metrics).map { ledger =>
       new LedgerBackedIndexService(MeteredReadOnlyLedger(ledger, metrics), participantId) {
         override def getLedgerConfiguration(): Source[v2.LedgerConfiguration, NotUsed] =
-          // FIXME(JM): This is broken. We should not use ReadService in Ledger API Server,
-          // The indexer should on start set the default configuration.
-          readService.getLedgerInitialConditions().map { cond =>
-            v2.LedgerConfiguration(cond.config.timeModel.minTtl, cond.config.timeModel.maxTtl)
-          }
+          // FIXME(JM): The indexer should on start set the default configuration.
```

Comment:
@dajmaki Do you think we can remove this FIXME?

Reply:
As long as we track this properly and fix it and the other FIXME that got removed above.

Reply:
OK, will keep it.